tests/end2end/test_keras_converter.py

```diff
@@ -19,8 +22,8 @@ def __init__(self, alpha=1.0, beta=1.0, **kwargs):
     def build(self, input_shape):
         super(ScaledTanh, self).build(input_shape)
 
-    def call(self, x):
-        return self.alpha * K.tanh(self.beta * x)
+    def call(self, inputs, **kwargs):
```

Comment:
**kwargs: is it necessary?

Reply:
compliant to keras interface to kill PEP warning.